src/app/pages/credentials/certificates-dash/certificates-dash.component.ts

```diff
@@ -67,7 +65,7 @@ export class CertificatesDashComponent implements OnInit {
     });
     this.systemGeneralService.getUnsignedCertificates().pipe(untilDestroyed(this)).subscribe((res) => {
       res.forEach((item) => {
-        this.unsignedCertificateAuthorities.push(
+        this.signCsrFieldConfSelect.options.push(
           { label: item.name, value: item.id },
         );
       });
```

Comment:
I think we don't need call this subscribe on init because it would called whenever clicking checkmark icon.